webapp/channels/src/components/youtube_video/youtube_video.test.tsx

```diff
@@ -91,4 +91,48 @@ describe('YoutubeVideo', () => {
 
         expect(wrapper.find(ExternalImage).prop('src')).toEqual('linkUrl');
     });
+
+    test('should match init snapshot (Shorts)', () => {
+        const store = mockStore(initialState);
+        const props = {
+            ...baseProps,
+            link: 'https://www.youtube.com/shorts/2oa5WCUpwD8',
+        };
+        const wrapper = mount(
+            <Provider store={store}>
+                <YoutubeVideo {...props}/>
+            </Provider>,
+        );
+        expect(wrapper).toMatchSnapshot();
+        expect(wrapper.find(ExternalImage).prop('src')).toEqual('linkForThumbnail');
+        expect(wrapper.find('a').text()).toEqual('Youtube title');
+        expect(wrapper.find('.video-shorts').exists()).toBe(true);
+        expect(wrapper.find('.video-shorts-expanded').exists()).toBe(false);
+    });
+
+    test('should match snapshot for playing state (Shorts)', () => {
+        const wrapper = shallow(
+            <YoutubeVideo
+                {...baseProps}
+                link={'https://www.youtube.com/shorts/2oa5WCUpwD8'}
+            />,
+        );
+        wrapper.setState({playing: true, shortsExpanded: false});
```

Comment:
We are planning to move all our components to function components, so it would be great if we can avoid tests that relay on "class components behavior" (in this case, the `setState`).

You may want to consider fully rendering the component with `renderWithContext`, and doing the actions that will set the component in this state.